src/h5web/visualizer/utils.test.ts

```diff
@@ -1,50 +1,59 @@
 import { getSupportedVis } from './utils';
 import { Vis } from '../visualizations';
-import { mockMetadata } from '../providers/mock/data';
+import {
+  compoundType,
+  intType,
+  makeAttr,
+  makeStrAttr,
+  scalarShape,
+} from '../providers/mock/data';
+import {
+  makeMyDataset,
+  makeMyGroup,
+  makeMyNxDataGroup,
+  makeMySimpleDataset,
+} from '../test-utils';
 
 describe('Visualizer utilities', () => {
   describe('getSupportedVis', () => {
     it('should return supported visualizations', () => {
-      const supportedVis = getSupportedVis(
-        mockMetadata.datasets?.raw,
-        mockMetadata
-      );
+      const datasetRaw = makeMyDataset('raw', scalarShape, compoundType);
+      const supportedVis = getSupportedVis(datasetRaw);
 
       expect(supportedVis).toEqual({ supportedVis: [Vis.Raw] });
     });
 
     it('should not include Raw vis if any other visualization is supported', () => {
-      const supportedVis = getSupportedVis(
-        mockMetadata.datasets?.oneD,
-        mockMetadata
-      );
+      const datasetInt1D = makeMySimpleDataset('dataset', intType, [5]);
```

Comment:
I might use entities from `data.ts` again later once I'll have refactored the mock data provider.